empress/vistrans/ReconRenderer.py

```diff
@@ -0,0 +1,418 @@
+import RenderClasses
+import tkinter as tk
+
+
+NODE_RADIUS = 5
+TREE_OFFSET = [20, 30]
+PARASITE_TREE_OFFSET = 15
+HOST_TREE_COLOR = 'green'
+PARASITE_TREE_COLOR = 'red'
+WINDOW_WIDTH = 1000
+WINDOW_HEIGHT = 1000
+WINDOW_NAME = "reconciliation"
+
+
+def computeHostNodeLogicalPositions(host_tree):
+    """ 
+    Sets the logicalRow and logicalCol values of each Node in host_tree.
+    Assumes that each host Node has its order set already and this function
+    uses those values and structure of the tree to set the logicalRow and logicalCol
+    :param host_tree:  A Tree object representing the host tree
+    :return None
+    """
+    #set logical cols
+    for node in host_tree.allNodes:
+        node.logicalCol = node.order
+
+    #sets logical row values for leaves in the order they appear in host_tree.leaves
+    logical_row_counter = 0
+    for leaf in host_tree.leaves:
+        leaf.logicalRow = logical_row_counter
+        logical_row_counter += 1
+    
+    #helper function to assign row values, postorder traversal
+    computetHostNodeLogicalPositions_helper(host_tree.rootNode)
+
+    
+def computetHostNodeLogicalPositions_helper(node):
```

Comment:
please use snake case for function names :)

Reply:
Hey could you make me a collaborator to this repo?

Reply:
Yes.